src/scripts/start/start.ts

```diff
@@ -9,8 +9,8 @@ import webpackDevServer from "webpack-dev-server";
 import webpackConfig from "../../webpack/webpack.config";
 import getConfigToUse from "../../common/getConfigToUse";
 
-const configToUse = getConfigToUse<webpack.Configuration>("start.js", webpackConfig);
-console.log(configToUse.isCustom ? `Found custom start config at ${configToUse.customPath}` : "Using default start config");
+const configToUse = getConfigToUse<webpack.Configuration>("start", webpackConfig);
+console.log(configToUse.isCustom ? `Found custom start config` : "Using default start config");
```

Comment:
You are repeating this same log in every script and adding `isCustom` attr to `getConfigToUse` method which is a bit confusing because I would never expect that it will return high level object with config instead of config itself. Maybe it will be better to add this log to `getConfigToUse`?

Reply:
Yes, I'm planning to refactor this entirely today, there is too much repetition in the scripts.

Reply:
To make logs in make script you need to knows there inner details of config resolver repeating the logic which is anti pattern. It should be either in `getConfigToUse` or in separate log util that will be checking if there is a custom config or not and will be utilized by both `getConfigToUse` and scripts but still you will be repeating your logging code.